aioxmpp/avatar/xso.py

```diff
@@ -24,9 +24,32 @@
 
 from aioxmpp.utils import namespaces
 
+from ..stanza import Presence
+
+
 namespaces.xep0084_data = "urn:xmpp:avatar:data"
 namespaces.xep0084_metadata = "urn:xmpp:avatar:metadata"
 
+namespaces.xep0153 = "vcard-temp:x:update"
+
+
+class VCardTempUpdate(xso.XSO):
+    """
+    The vcard temp update note as per :xep:`0153`
```

Comment:
s/temp //, it isn’t useful information to the reader.

Reply:
and s/note/node/